### DISCUSSION TITLE
A decidir el futuro de la aplicación 🫡

### DISCUSSION THREAD

**Discussion by 2021030063** [Centinela-RR/Custodes#5] [Encuestas]
# **Invocando a todos!**

@Centinela-RR/db @Centinela-RR/design @Centinela-RR/dev @Centinela-RR/documentacion  @Centinela-RR/investigadores 

Esto es en referencia a la discusión #4, en donde se habla de cambiar la dirección a tomar de la aplicación.

La idea es:
Alterar la aplicación para que el público pueda tenerla abierta a todo rato al nivel del volante (como Google maps), que tenga un botón de S.O.S. para casos de secuestros; que envíe un mensaje con la ubicación si quieren a algún contacto guardado de confianza. 
Solo para que mínimo sepan dónde desapareció el individuo o algo asi. 
Básicamente, una aplicación para avisar donde hay un conflicto; ya que a la mayoría de conductores les vale si hay un poste chocado o una fuga, y si existe Google ya avisa tráfico lento.

Cómo se implementaría?
- Agregando un botón junto al botón de reportar incidentes que sea de SOS

Qué implicaría?
- Eliminar el aspecto de reportes de servicios primarios e incendios para enfocarse únicamente en inseguridad

Cambiaría fundamentalmente el funcionamiento de la aplicación como tal?
- Sí, eliminaría un aspecto importante de ella

Editado:
Si no hay necesidad de eliminar y podemos ocultar cosas que no sean útiles o necesarias con opciones en configuración o similar, por qué no? ;D

**Comment by Yoel1515** [comment]
Mmmm primero aver que piensan los demás y voto

**Comment by JW33D** [comment]
Que arda Troya jajaj

**Comment by Oliasaew** [comment]
Pienso que sería un muy buen agregado a la aplicación inicial, sin embargo, ya existen muchas otras aplicaiciones que lanzan la ubicación a tiempo real, incluso whatsapp (donde pueden estar viendo tus contactos tu ubi a tiempo real) y life 360 (que ya varios compañeros tienen) , no sé si aguirre vaya a considerarlo como algo nuevo, mientras que la aplicación inicial es algo que aguirre ya aceptó y es rentable para vender o ofrecer a gobiernos, no existen muchas otras aplicaciones en las que se puedan hacer reportes sobre lo que tenemos en mente
